src/main/java/appeng/container/implementations/ContainerDrive.java

```diff
@@ -29,7 +29,7 @@
 public class ContainerDrive extends AEBaseContainer
 {
 
-	final TileDrive drive;
+	private final TileDrive drive;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "drive" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)